frontend/src/components/MainSearch/MainSearch.js

```diff
@@ -16,6 +17,16 @@ class MainSearch extends Component {
     this.togglePanel = this.togglePanel.bind(this);
   }
 
+  componentDidMount() {
+    const cookies = new Cookies();
+    if (cookies.get('visit') !== 'set') {
+      this.setState({
+        open: true,
+      });
+      cookies.set('visit', 'set', { path: '/' });
```

Comment:
NitPick: I think we should change the cookie naming a little. Something like `cookies.set('rlc', 'visited', { path: '/' });` would be a little more descriptive.